articles/20240819_jupyter_notebook_guide.md

```diff
@@ -0,0 +1,257 @@
+4. When you execute code, it's sent to the kernel for processing.
+5. Results are returned to the browser for display.
+
+This architecture allows for interactive coding sessions where you can see results immediately and iterate quickly.
+
+**Key Point:** Jupyter Notebooks (.ipynb files) contain both the input and output of your interactive sessions, making them excellent for sharing and reproducing work.
+
+### Sharing Notebooks
+
+You can share your notebooks in two main ways:
+1. Download the notebook as a raw .ipynb file, which others can open on their own Jupyter server.
+2. Share the URL directly with users who have access to the same server. (Note: To enable real-time collaboration via URL sharing, Jupyter must be run using the --collaborative flag.)
+
+## Getting Started with Jupyter Notebook in Daytona
+
+### Installation Guide
+
+Daytona makes it easy to set up Jupyter Notebook in your development environment. Here's a detailed guide on how to get started:
+
+1. **System Requirements**:
+   - Web browser.
+   - Docker installed on your machine.
+
+2. **Installation Steps**:
+   - Log in to your Daytona account
+   - Create a repo with the .devcontainer folder and the devcontainer.json file
+   - Create a new project or open an existing one
+   - Daytona will automatically pull and set up the dev container
+   - Voila! You have a Jupyter Notebook environment in Daytona
+Daytona supports multiple IDEs, including Visual Studio Code and JetBrains IDEs. While this guide focuses on Jupyter Notebook, you can use these other IDEs with Jupyter extensions for a more integrated development experience.
+
+### Workspace Management for Jupyter Notebook Projects
+
+Daytona's workspace management features enhance your Jupyter Notebook workflow:
+
+1. **Persistent Workspaces**: Your Jupyter Notebook environment persists between sessions, allowing you to pick up right where you left off.
+2. **Resource Management**: Easily adjust computational resources allocated to your Jupyter environment as your project needs change.
+3. **Environment Replication**: Quickly replicate your Jupyter setup for team members or different branches of your project.
+
+### Version Control Integration
+
+Daytona seamlessly integrates with various Git providers (GitHub, GitLab, and Bitbucket), offering robust version control for your Jupyter notebooks:
+
+1. **Automatic Syncing**: Changes in your notebooks are automatically tracked and can be committed directly from the Daytona interface.
+2. **Branch Management**: Create and switch between branches to manage different versions of your notebooks.
+3. **Collaboration**: Easily share your notebook changes with team members through pull requests and code reviews.
+
+### Sharing and Collaboration
+
+Daytona enhances the sharing capabilities of Jupyter Notebook:
+
+1. **Team Access**: Grant team members access to your Jupyter environment, allowing for real-time collaboration.
```

Comment:
this article is a mess, please understand what is Daytona open source and rewrite the article or close the PR